ci-operator/step-registry/openshift/e2e/aws/ccm/openshift-e2e-aws-ccm-workflow.yaml

```diff
@@ -6,6 +6,7 @@ workflow:
     - ref: ccm-conf-apply-feature-gate
     test:
     - ref: openshift-e2e-test
+    - ref: machine-api-e2e-test
```

Comment:
Is it preferable to run our E2E suite as a separate workflow? That way it would get a separate status check as it normally does?